### PR TITLE
Error if quarto with old bootstrap

### LINKED ISSUE #2795: Improve error message when using Quarto vignette with BS3
#2790 

Or to ignore since BS3 will be deprecated at one point anyway? :sweat_smile:

### PR DESCRIPTION
**Pull request by jayhesselberth**
Fixes #2795.

### CODE REVIEW

File: R/build-quarto-articles.R [hunk 1]
```diff
@@ -1,5 +1,8 @@
 build_quarto_articles <- function(pkg = ".", article = NULL, quiet = TRUE) {
   pkg <- as_pkgdown(pkg)
+  if (pkg$bs_version < 5) {
```
Comment (jayhesselberth) [inline R/build-quarto-articles.R:3]: Do I also need to check for `NULL`?

Comment (maelle) [reply R/build-quarto-articles.R:3]: is it NULL at this stage if nothing is set in the config file?

File: R/build-quarto-articles.R [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+    cli::cli_abort("Quarto articles require Bootstrap 5.", call = NULL)
```
Comment (maelle) [inline R/build-quarto-articles.R:4]: maybe also link to https://pkgdown.r-lib.org/articles/customise.html#getting-started

**Comment by maelle** [comment]
Thank you :smile_cat:

**Pull request merged**